cherrypy/wsgiserver/__init__.py

```diff
@@ -1939,7 +1939,11 @@ def start(self):
             self.software = "%s Server" % self.version
 
         # Select the appropriate socket
-        if isinstance(self.bind_addr, six.string_types):
+        self.socket = None
+        if os.environ.get('LISTEN_PID', None):
```

Comment:
Why set default value to `None`? It's not necessary.
Moreover, [`os.getenv('LISTEN_PID')`](https://docs.python.org/2/library/os.html#os.getenv) would be more verbose.